### ISSUE TITLE
Low Test Accuracy

### ISSUE THREAD

**Issue by Vigelos** [sail-sg/VGT#4]
After downloading all the required files for nextqa, I executed the command :
```shell
./shells/next_test.sh 0
``` 
but found that the accuracy is only 19.47%.

And when I removed the line of "pretrain_path" in **next_test.sh** and executed the command aforementioned, I got the accuracy 20.04%. 

So I wander whether fine-tuning is required for the given model "VGT_B5" on nextqa.

**Comment by doc-doc** [comment]
Hi, no further fine-tuning is needed for VGT_B5.

**Comment by doc-doc** [comment]
Please check if you have set-up the repo. correctly.

**Comment by Vigelos** [comment]
thanks for your clarification, I will check all my steps again :)
